application-ckeditor-plugins/src/main/resources/xwiki-macro/macroSelector.js

```diff
@@ -152,6 +186,8 @@ define('macroSelector', ['jquery', 'modal', 'l10n!macroSelector'], function($, $
     }
     // Add separator before "Other" category.
     categoryFilter.find('.macro-category[data-category=""]').before(separator);
+    // Add separator before "Not installed" category.
```

Comment:
I wouldn't add the separator if the 'Other' category is already separated. I mean, I'd group the 'Other' and 'Not installed' categories. I have the feeling we're showing too many separators otherwise.

Reply:
I'm not sure about this one, "Other" category feels much closer to other installed macro categories than to the "Not installed" category, so it does not make much sense to group them together.